windows/src/desktop/kmshell/main/Keyman.System.RemoteUpdateCheck.pas

```diff
@@ -375,4 +218,44 @@ function TRemoteUpdateCheck.DoRun: TRemoteUpdateCheckResult;
    KL.Log(LogMessage);
  end;
 
+function ConfigCheckContinue: Boolean;
```

Comment:
The function name doesn't tell me much. Would `CheckForUpdatesIsDue` or `ShouldCheckForUpdates` be a better name?